target_finder/preprocessing.py

```diff
@@ -82,6 +81,33 @@ def find_blobs(image, min_width=20, max_width=100, limit=100, padding=20):
     return blobs
 
 
+def _is_shape_like_blob(blob, min_width, max_width):
+    """Verify that this could be a blob containing a shape"""
+    width = blob.width
+    height = blob.height
+
+    # check provided restrictions
+    if min(width, height) < min_width or max(width, height) > max_width:
+        return False
+
+    # must be closed cnt
+    # if not blob.has_mask:
```

Comment:
What was this check originally for?

Reply:
The whole thing or just the commented out part?

Reply:
Yeah the commented out part

Reply:
Ah yeah the close contour story is a dependency for that, I told Shrivu to just comment it out for now until that gets sorted. It’s just a check to make sure it’s a fully closed contour that we can apply a mask to.

Reply:
@sshh12 Can you leave a TODO comment on there saying to uncomment once the close contour story is done?